Wanikani/Burn Bell/userscript.js

```diff
@@ -82,7 +82,7 @@
 
     window.addEventListener('didChangeSRS', (e) => {
         const srs = e.detail.newLevelText
-        if (/burn/i.test(srs) && settings.bell !== 'disabled') bellSound.play()
+        if (/burn/i.test(srs) && settings.bell !== 'disabled') bellSound.pause() || (bellSound.currentTime = 0) || bellSound.play()
```

Comment:
This is more just for my knowledge but how is this line working? How is it different from just using `;` or `,`?

To the actual review part, I see that Kumi has addressed this separately in Reorder Omega with the following code:
```ts
function install_burn_bell(): void {
    window.addEventListener('didChangeSRS', (e: any) => {
        const srs = e.detail.newLevelText
        if (!/burn/i.test(srs) || settings.burn_bell === 'disabled') return
        burn_bell_audio.load() // Stop if already playing
        burn_bell_audio.play()
    })
}
```
Here `burn_bell_audio` would be `bellSound`. I find this more clear so perhaps this could be done instead?

Reply:
Yeah, the user experience is the same, though [load()](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/load), from what I understand, ends up causing another network call.
> This method is generally only useful when you've made dynamic changes to the set of sources available for the media element.

The reason I went with the `||` chaining was simply to condense the code, keep the `if` statement inline, and avoid semicolons (since Kumirei doesn't use them from what I could tell). I wouldn't mind changing it to a bracketed `if` with either `;` or as multiple line-separated statements, if clarity is important. Or, I guess, on the off chance that `pause()` changes in the future to possibly return a truthy value.

Reply:
Ahh, I see. I kind of forget that functions always return something, even if that something is `undefined`. Makes more sense now, but yeah, I prefer the way it's done in Omega. Thanks for the tidbit about the network call too.